README.md

```diff
@@ -1 +1,95 @@
-# FIrst-Microverse
\ No newline at end of file
+ # [First-Microverse-Hello-Project]
+
+#### First-Microverse-Hello-Project is a beautiful project that prints Hello Microverse on the screen with a red text color.
+
+## 🛠 Built With
+
+### HTML
+### CSS
+### Linters
+
+
+## Key Features
+
+### [Visual Studio Code]: This is environment in which the application is built.
+### [HTML] : This is a Markup Language use to achieve the project
+### [Linters] : This is a tool used in the project to avoide coding error and professionality
+
+
+
+## 💻 Getting Started
+
+### You will need the following tools to get started with the project
+
+### 1. Installation of Git
+### 2. Installation of NPM
+### 3. A working browser
+### 4. Internet connection
+### 5. A code editor like Visual Studio Code
+### 6. Installation of Live Server plugging in Visual Studio Code
+### 7. Linters
+
+### To get a local copy up and running, follow these steps:
+
+## Prerequisites:
+
+#### In order to run this project you need: Visual studio Code, Internet, GitHub account.
+
+## Setup:
+
+#### Clone this repository to your desired folder: Clone this repo with git clone command.
+
+## Install:
+
+#### Install this project with: NPM install, Create a .github/workflow folder and add linter.yml in it
+#### add .hintrc file, also add .stylelintrc.json file.
+
+## Usage:
+
+#### To run the project: right click on your html environment and click on run with live server.
+
+## Run Linter tests:
+
+#### To run linter tests, run the following command: npx stylelint "**/*.{css,scss}" 
+
+## Deployment
+#### You can deploy this project using: Host services
+
+
+
+# 👥 Authors
+
+#### Toyyib Omobolaji Bello
+
+# 👤 Author1
+
```

Comment:
Great Readme. However, let's make it even better 💪. I believe that if you take out  ```Author1``` and replace that with your name, your this section would look more projessional. It would make the section look something like this:

``` 
# 👥 Author

#### 👤 Toyyib Omobolaji Bello